universal-login-react/src/core/utils/getErc721Tokens.ts

```diff
@@ -0,0 +1,52 @@
+import {fetch} from '@unilogin/commons';
+
+interface IBasicToken {
+  tokenID: string;
+  contractAddress: string;
+}
+
+const OPENSEA_API_HTTP = 'https://api.opensea.io/api/v1/asset';
+
+const ETHERSCAN_API_HTTP = 'http://api.etherscan.io/api?module=account&action=tokennfttx&startblock=0&endblock=999999999&sort=asc&apikey=51GC96WCST6YU1BKYWGGX3PIT21UJPTFTE';
+
+export interface IToken {
+  id: string;
+  name: string;
+  image: string;
+  description: string;
+  backgroundColor: string;
+  tokenName: string;
+  externalink: string;
+}
+
+export const getErc721Tokens = async (walletAddress: string, currentTokens: IToken[] = [], network = 'mainnet') => {
+  let etherscanApi = ETHERSCAN_API_HTTP;
+  let openseaApi = OPENSEA_API_HTTP;
+  if (network === 'rinkeby') {
```

Comment:
What will happen if the network is kovan

Reply:
It is not implemented yet, I implemented only mainnet and rinkeby

Reply:
So maybe we should throw some specific error?